internal/clients/config/base.go

```diff
@@ -23,6 +23,20 @@ func newBaseConfigFromSDK(d *schema.ResourceData, version string, esKey string)
 		Header:    http.Header{"User-Agent": []string{userAgent}},
 	}
 
+	if elasticDataRaw, ok := d.GetOk("elasticsearch"); ok {
```

Comment:
This should be added in `config/elasticsearch` instead. It's not supported by the Kibana API.